packages/kbn-plugin-helpers/tasks/build/create_build.js

```diff
@@ -65,6 +70,30 @@ function parseTsconfig(pluginSourcePath, configPath) {
   return config;
 }
 
+// transpile with babel
+async function transpileWithBabel(srcGlobs, buildRoot, presets) {
+  await asyncPipeline([
```

Comment:
Pretty sure the first argument can't be an array, but each stream should be a separate argument

Reply:
@spalger even not documented it would work https://github.com/nodejs/node/blob/master/lib/internal/streams/pipeline.js#L68

but maybe it's better to change to the documented way